docs/source/error_code_list.rst

```diff
@@ -430,6 +430,65 @@ Example:
     # Error: Incompatible types (expression has type "float",
     #        TypedDict item "x" has type "int")  [typeddict-item]
     p: Point = {'x': 1.2, 'y': 4}
+    
+Check TypedDict Keys [typeddict-unknown-key]
+--------------------------------------------
+
+When constructing a ``TypedDict`` object, mypy checks whether the definition
+contains unknown keys.
+
+For convenience's sake, mypy will not generate an error when a ``TypedDict``
+has extra keys if it's passed to a function as an argument.
+
+However, it will generate an error when these are created.
+
+Example:
```

Comment:
```suggestion
When constructing a ``TypedDict`` object, mypy checks whether the definition
contains unknown keys. For convenience's sake, mypy will not generate an error
when a ``TypedDict`` has extra keys if it's passed to a function as an argument.
However, it will generate an error when these are created. Example:
```